### DISCUSSION TITLE
Webassembly template for F#

### DISCUSSION THREAD

**Discussion by 3xau1o** [mono/SkiaSharp#2202] [Show and tell]
Managed to get a barebones Skia text rendering using GTKSharp3 https://github.com/mindlid/gtk3SkiaFS

How would this be translated to blazor using F#, there is a blazor [SKCanvasView](https://docs.microsoft.com/en-us/dotnet/api/skiasharp.views.blazor.skcanvasview?view=skiasharp-views-2.88), maybe getting it running with [bolero](https://fsbolero.io)?

**Comment by mattleibow** [comment]
If you are looking at Blazor WASM then there is this https://www.nuget.org/packages/SkiaSharp.Views.Blazor

**Reply by 3xau1o** [reply]
i was looking for a basic working template with F#, seems like almost everything from Blazor is XAML based.

**Reply by mattleibow** [reply]
Blazor is not XAML but rather razor which is like HTML on vitamins. Blazor is basically web technology. Not even sure Blazor supports F#.

Maybe you need to create a new view on the Bolero framework? Not sure what that would look like, but with regards to SkiaSharp, all I do is add an HTML canvas to the screen and the rest is C# and JS code.

 - https://github.com/mono/SkiaSharp/blob/main/source/SkiaSharp.Views.Blazor/SkiaSharp.Views.Blazor/SKGLView.razor.cs
 - https://github.com/mono/SkiaSharp/blob/main/source/SkiaSharp.Views.Blazor/SkiaSharp.Views.Blazor/wwwroot/SKHtmlCanvas.ts

I think the two frameworks might be fairly different, so a new view/element is going to be needed.

**Reply by 3xau1o** [reply]
thanks, very likely to require an F# view written from scratch
